src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -3307,14 +3307,6 @@ encode %{
   %}
 
 
-  enc_class enc_cmove_imm(iRegIdst dst, flagsRegSrc crx, immI16 src, cmpOp cmp) %{
-    Label done;
-    assert((Assembler::bcondCRbiIs1 & ~Assembler::bcondCRbiIs0) == 8, "check encoding");
-    // Branch if not (cmp crx).
-    __ bc(cc_to_inverse_boint($cmp$$cmpcode), cc_to_biint($cmp$$cmpcode, $crx$$reg), done);
-    __ li($dst$$Register, $src$$constant);
-    __ bind(done);
-  %}
```

Comment:
Now, you have a lot of empty lines! Also at other places where you removed code.